classes/tdx-signed-harden.inc

```diff
@@ -13,12 +13,18 @@
 # UBOOT_SIGN_ENABLE are set, by default.
 #
 def default_tdx_uboot_hardening_enable(d):
-    if (bb.utils.to_boolean(d.getVar('TDX_IMX_HAB_ENABLE')) and
+    if ('imx-generic-bsp' in d.getVar('MACHINEOVERRIDES_EXTENDER').split(':') and
+        bb.utils.to_boolean(d.getVar('TDX_IMX_HAB_ENABLE')) and
+        bb.utils.to_boolean(d.getVar('UBOOT_SIGN_ENABLE'))):
+        return True
+    if ('k3' in d.getVar('OVERRIDES').split(':') and
+        bb.utils.to_boolean(d.getVar('TDX_K3_HSSE_ENABLE')) and
         bb.utils.to_boolean(d.getVar('UBOOT_SIGN_ENABLE'))):
         return True
     return False
 
 TDX_UBOOT_HARDENING_ENABLE ?= "${@'1' if default_tdx_uboot_hardening_enable(d) else '0'}"
+TDX_UBOOT_HARDENING_ENABLE:verdin-am62-k3r5 = "0"
```

Comment:
Out of curiosity I know this refers to the "other" machine configuration for the am62 and not the "real" one of `verdin-am62`. 

I assume you explicitly set this to 0 here for the "other" configuration due to weird interactions?

Reply:
Yes, I didn't want the hardening enabled when compiling U-Boot for that other machine simply because it wasn't designed for it. As I mentioned in another message, this might leave some uncovered attack surface. If other protections are needed, I imagine they would be different from what we have today and it would require a new implementation. I intended to capture this in a separate ticket.

Reply:
@jsrc27 Is it okay to resolve this thread?